crates/yas_core/src/contracts/yas_pool.cairo

```diff
@@ -199,6 +203,16 @@ mod YASPool {
             self.token_1.read()
         }
 
+        fn slot_0(self: @ContractState) -> (FixedType, i32, u8, bool) {
+            let slot_0 = self.slot_0.read();
+            (slot_0.sqrt_price_X96, slot_0.tick, slot_0.fee_protocol, self.unlocked.read())
+        }
```

Comment:
This implementation will collide with the one made in PR #135 :
`fn slot_0(self: @ContractState) -> Slot0 {
            self.slot_0.read()
        }`

Reply:
✔️